app/models/course/assessment/question/text_response_comprehension_solution.rb

```diff
@@ -33,6 +35,11 @@ def strip_whitespace_solution
     solution.each(&:strip!)
   end
 
+  def convert_solution_to_lemma
+    solution_lemma.clear
+    solution.each { |s| solution_lemma.push(WordNet::Synset.morphy_all(s).first || s) }
```

Comment:
Suggest that we have a separate service here which deals with conversion of lemmas. It will be 1-2 lines only but it allows us to change the lemma-tizer easily next time. Can be used by the autograder as well.

Reply:
Okay, will do.